lib/v2/freebuf/index.js

```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { type = 'web' } = ctx.params;
+
+    const fapi = 'https://www.freebuf.com/fapi/frontend/category/list';
+    const baseUrl = 'https://www.freebuf.com';
+    const rssLink = `${baseUrl}/articles/${type}`;
+
+    const options = {
+        headers: {
+            referer: 'https://www.freebuf.com',
+            accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7',
+        },
+        searchParams: {
+            name: String(type),
```

Comment:
```suggestion
            name: type,
```
`type` is string by default no need to type cast it again.

Reply:
`String()` was removed. (By the way, the `String()` was added by command `yarn format`, that's why I kept it early.)